tests/RadLineCooling.in

```diff
@@ -23,4 +23,6 @@ radiation.dust_gas_interaction_coeff = 1e-20
 
 do_reflux = 0
 do_subcycle = 0
-suppress_output = 1
\ No newline at end of file
+suppress_output = 1
+
+plotfile_interval = 10000000
```

Comment:
If you want to turn off output, you can set `plotfile_interval = -1`.

Reply:
I know. I want one plotfile output to show the metadata. However, for some reason, it didn't work as expected. The code did not save any pltfile output.

Reply:
The `plotfile_interval` can be hardcoded in the *.cpp problem file. Then it overrides any settings in the input file.